Makefile

```diff
@@ -131,7 +131,10 @@ test-unit: generate manifests ## Run unit tests only.
 
 .PHONY: test-integration
 test-integration: generate manifests build-ebpf ## Run integration tests.
-	EBPF_BUILD_DIR=$(EBPF_BUILD_DIR) ANTIMETAL_BPF_PATH=$(EBPF_BUILD_DIR) go test -tags integration -p 1 ./... -v -timeout 60s
+	EBPF_BUILD_DIR=$(EBPF_BUILD_DIR) ANTIMETAL_BPF_PATH=$(EBPF_BUILD_DIR) go test -tags integration \
+		./pkg/kernel \
+		./pkg/ebpf/core \
+		-v -timeout 60s
```

Comment:
This seems kind of dangerous since integration tests in other packages could silently fail and we wouldn't catch like what happened with the collector integration tests